app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,441 @@
+<!--
+  ~ Copyright 2019 ACINQ SAS
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+  <string name="app_name">Phoenix (Testnet)</string>
+
+  <!-- //////////////// system notification channels //////////////// -->
+
+  <string name="notification_channels_watcher_title">Sledování na pozadí</string>
+  <string name="notification_channels_watcher_desc">Upozorní vás pokud je potřeba spustit aplikaci. Má velkou prioritu.</string>
+
+  <!-- //////////////// init wallet / automated wallet creation //////////////// -->
+
+  <string name="autocreate_generating">Vytváření peněženky…</string>
+  <string name="autocreate_error">Nepodařilo se vytvořit peněženku.\nDůvod: %1$s</string>
+
+  <!-- //////////////// init wallet / automated wallet creation //////////////// -->
+
+  <string name="restore_disclaimer_message">Neimportujte seed, který nebyl vytvořen v této aplikaci. \n\nZároveň se ujistěte že nemáte jinou Phoenix peněženku se stejným seedem.</string>
+  <string name="restore_disclaimer_checkbox">Ano, chápu.</string>
+  <string name="restore_disclaimer_next">Další</string>
+  <string name="restore_instructions">Seed vaší peněženky je seznam 12 anglických slov. Heslo peněženky není podporováno.</string>
+  <string name="restore_mnemonics_label">Použijte klávesnici k zadání seedu:</string>
+  <string name="restore_in_progress">Obnovování peněženky…</string>
+  <string name="restore_import_button">Importovat</string>
+  <string name="restore_error">Tento seed je neplatný a nelze ho importovat.\n\nProsím zkuste to znovu</string>
+
+  <!-- //////////////// Pin dialog //////////////////// -->
+
+  <string name="pindialog_title_default">Prosím zadejte váš PIN</string>
+  <string name="pindialog_pwd_placeholder">●●●●●●</string>
+  <string name="pindialog_error">PIN kód musí být 6 ciferné číslo.</string>
+  <string name="pindialog_error_donotmatch">PIN kódy nesouhlasí.</string>
+
+  <!-- //////////////// Biometric dialog //////////////////// -->
+
+  <string name="biometricprompt_title">Peněženka je uzamčena</string>
```

Comment:
Peněženka je uzamčena.

Reply:
If the difference is the dot at the end, then that's on purpose.